src/Drag_Drop.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-restricted-syntax */
+// eslint-disable-next-line import/prefer-default-export
+export function dragDrop() {
+  const target = document.getElementById('ListContainer');
+  const items = target.getElementsByTagName('li');
+  let current = null;
+  for (const i of items) {
+    i.draggable = true;
+    // eslint-disable-next-line func-names
+    i.addEventListener('dragstart', function () {
+      current = this;
+    });
+
+    i.addEventListener('dragover', (evt) => {
+      evt.preventDefault();
+    });
+    // eslint-disable-next-line func-names
+    i.addEventListener('drop', function () {
+      if (this !== current) {
+        let currentpos = 0; let
+          droppedpos = 0;
+        // eslint-disable-next-line no-plusplus
+        for (let it = 0; it < items.length; it++) {
```

Comment:
- [ ] Good job implementing the previously required changes. There are a few tweaks to be made here:
- The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.